src/drivers/bitbucket_cloud.js

```diff
@@ -21,9 +21,19 @@ class BitBucketCloud {
     const { project_path } = this;
     const { commit_sha, report } = opts;
 
-    // Print some variables
-    const endpoint = `/repositories/${project_path}/commit/${commit_sha}/comments/`;
-    const body = JSON.stringify({ content: { raw: report } });
+    // Get the PR ID #
+    const pr_endpt = `/repositories/${project_path}/commit/${commit_sha}/pullrequests`;
+    const pr_out = await this.request({ endpoint: pr_endpt });
+    const pr_id = pr_out.values[0].id;
+
+    // Append a watermark to the report with a link to the commit
+    const commit_url = `https://bitbucket.org/${project_path}/commits/${commit_sha}`;
+    const commit_link = `[Go to commit.](${commit_url})`;
```

Comment:
better 

```js
const commit_link = commit_sha.substr(0, 7);
const long_report = `${commit_link}\n  ${report}`;
```

shas are normally generates links so we dont need to know the url.
Also I would put it at the beginning

Reply:
thanks for the suggestion!